server/eclipse-project/src/test/java/au/org/democracydevelopers/corla/workflows/Workflow.java

```diff
@@ -56,16 +48,8 @@
 import org.apache.log4j.LogManager;
 import org.apache.log4j.Logger;
 import org.apache.http.HttpStatus;
-import org.testcontainers.containers.PostgreSQLContainer;
-import org.testcontainers.ext.ScriptUtils;
-import org.testcontainers.jdbc.JdbcDatabaseDelegate;
 import org.testng.annotations.BeforeClass;
-import us.freeandfair.corla.math.Audit;
-import us.freeandfair.corla.model.CVRContestInfo;
-import us.freeandfair.corla.model.CastVoteRecord;
-import us.freeandfair.corla.model.CastVoteRecord.RecordType;
```

Comment:
I'm guessing the changes in this file are just because you're now loading the data via an SQL file instead of manually adding it?